### PR TITLE
[Playwright] Add integration tests

### PR DESCRIPTION
**Pull request by omacranger**

### CODE REVIEW

File: .github/workflows/playwright-integration.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: playwright (integration test)
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'visual-js/visual-playwright/**'
+      - .github/workflows/js-build.yml
```
Comment (FriggaHel) [inline .github/workflows/playwright-integration.yml:9]: Is that a left-over or an actual prerequisite ? I guess the 2nd, but prefer to be sure.

File: visual-js/visual-playwright/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+WORKDIR app
+
+RUN corepack enable
+
+COPY . ./
+
+RUN yarn install && npm run build --workspaces --if-present
+
+COPY ./visual-playwright/integration-tests/ ./integration-tests/
```
Comment (FriggaHel) [inline visual-js/visual-playwright/Dockerfile:11]: Isn't this useless ?

Hasn't `COPY . ./` already pushed that folder too ?

**Review by FriggaHel — APPROVED** [review]

File: .github/workflows/playwright-integration.yml (hunk 1 above)
Comment (omacranger) [reply .github/workflows/playwright-integration.yml:9]: Um, not a prerequisite per se, but changing of the JS Build system probably means we should run the integration tests for the affected tooling right?

File: visual-js/visual-playwright/Dockerfile (hunk 2 above)
Comment (omacranger) [reply visual-js/visual-playwright/Dockerfile:11]: True... But it pushes it into a nested folder. Let me see how it behaves if I remove that line.

File: .github/workflows/playwright-integration.yml (hunk 1 above)
Comment (FriggaHel) [reply .github/workflows/playwright-integration.yml:9]: True 👍

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**